lib/active_merchant/billing/gateways/airwallex.rb

```diff
@@ -139,8 +136,8 @@ def create_payment_intent(money, options = {})
         post = {}
         add_invoice(post, money, options)
         add_order(post, options)
-        post[:request_id] = options[:request_id]
-        post[:merchant_order_id] = options[:merchant_order_id]
+        post[:request_id] = "#{request_id(options)}_setup"
+        post[:merchant_order_id] = "#{merchant_order_id(options)}_setup"
```

Comment:
Airwallex will fail a transaction if it uses a `request_id` or `merchant_order_id` that has been used in a previous transaction. Because their typical flow involves two transactions (the `create_payment_intent` followed by some sort of `purchase`), we have chosen to append this suffix to the ids on the `create_payment_intent` call in order to make them different than the subsequent `purchase` call.

Reply:
This is very readable code. Thanks for adding this explanation, I always learn something when reviewing your PR's.